compiler/src/iree/compiler/Codegen/Common/MathTransformPass.cpp

```diff
@@ -177,6 +211,11 @@ class MathTransformPass final
         patterns,
         [target](StringRef name) { return predicateApprox(name, target); });
 
+    // Add device-lib implementation patterns
```

Comment:
style: implementation comments only required when they document something non-obvious (here there's a lot of `populate*Patterns` calls here doing the same thing, so there's nothing novel about this one that needs a comment)

```suggestion
```

https://google.github.io/styleguide/cppguide.html#Implementation_Comments